### PR TITLE
Created a Schema for the Goals model.

### PR DESCRIPTION
**Pull request by bryanwaine**
Created a new mongoose schema for the goals model. This closes issue #336.

### CODE REVIEW

**Comment by Asylcreek** [comment]
Please move this PR to the staging branch

File: models/goal.model.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        type: String,
+        required: true,
+        trim: true
+    },
+    description: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    monthlyGoal: {
```
Comment (Ajioz) [inline models/goal.model.js:14]: The weekly goal should be created as well

Comment (Ajioz) [reply models/goal.model.js:14]: Let the week goal comes before the monthly goal

Comment (bryanwaine) [reply models/goal.model.js:19]: Alright. I’ve don’t that already and created a pull request. Please review.

**Review by Ajioz — APPROVED** [review]

**Comment by Asylcreek** [comment]
Please make the PR to the staging branch instead

**Pull request closed without merging**